parasite/scripts/production/rnaseq/get_ena_studies_by_taxon_id.py

```diff
@@ -0,0 +1,38 @@
+import requests
+from ProductionUtils import *
+from dependencies import *
+import time
+
+from optparse import OptionParser
```

Comment:
May want to change this to argparse.
See: https://stackoverflow.com/questions/3217673/why-use-argparse-rather-than-optparse